src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -891,6 +891,11 @@ protected DynamoModel(IStartConfiguration config)
                             {
                                 if (loadedExtension is IExtension)
                                 {
+                                    if (loadedExtension is LinterExtensionBase loadedLinter)
+                                    {
+                                        loadedLinter.InitializeBase(this.LinterManager);
```

Comment:
what is this?

Reply:
This was added to mirror the init function for the LinterExtensionBase that is a few lines above.  The above lines handle internal extensions are `LinterExtensionBase`.  This init is for those that are loaded via the package manager